src/components/__tests__/MyCalendar.test.ts

```
@@ -58,24 +58,31 @@ describe('MyCalendar', () => {
     expect(wrapper.find('.week-view').exists()).toBe(false)
 
-    // Switch to week view
-    const weekBtn = wrapper.findAll('button').find(btn => btn.text() === 'Week')
-    await weekBtn?.trigger('click')
+    // Open the view dropdown
+    const dropdownBtn = wrapper.find('.custom-dropdown-btn')
+    await dropdownBtn.trigger('click')
+    // Find the week option and click it
+    const weekOption = wrapper.findAll('.custom-dropdown-menu li').find(li => li.text() === 'Week')
+    expect(weekOption).toBeTruthy()
+    await weekOption!.trigger('click')
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)